hackathon_contributions_new_data_format/Isensee_JCB2018/visualization/main.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import getDataToBePlotted
+
+DataFilePath = "https://raw.githubusercontent.com/LoosC/Benchmark-Models/" \
+               "hackathon/hackathon_contributions_new_data_format/" \
+               "Isensee_JCB2018/measurementData_Isensee_JCB2018.tsv"
+
+ConditionFilePath = "https://raw.githubusercontent.com/LoosC/" \
+                    "Benchmark-Models/hackathon/hackathon_contributions_" \
+                    "new_data_format/Isensee_JCB2018/" \
+                    "experimentalCondition_Isensee_JCB2018.tsv"
+
+VisualizationFilePath = "https://raw.githubusercontent.com/LoosC/"\
+                        "Benchmark-Models/visualization/hackathon_contributions"\
+                        "_new_data_format/Isensee_JCB2018/visualizationSpecific"\
+                        "ation_Isensee_JCB2018.tsv"
+
+# import measurement data
+measurement_data = pd.DataFrame.from_csv(
+        DataFilePath, sep="\t", index_col=None)
+
+# import experimental condition
+experimental_condition = pd.DataFrame.from_csv(
+        ConditionFilePath, sep="\t", index_col=None)
+
+# import visualization specification
+visualization_specification = pd.DataFrame.from_csv(
+        VisualizationFilePath, sep="\t", index_col=None)
+
+# Set Colormap
+cmap = ['#8c510a','#bf812d','#dfc27d','#f6e8c3','#c7eae5','#80cdc1','#35978f','#01665e']
```

Comment:
For later, it might be good to generate color maps automatically, depending on the number of categories we need.

E.g. https://matplotlib.org/users/colormaps.html, https://seaborn.pydata.org/tutorial/color_palettes.html

Reply:
I implemented the standard colorblind-map of seaborn now.
This has 10 different colours and the online colorbrewer can get up to 11 with the colobrlind option.
Would limiting it to say 3 colors, when only needing 3 make a difference?